### ISSUE TITLE
Testing conda installation

### ISSUE THREAD

**Issue by gputzel** [knights-lab/SHOGUN#19]
The `README` instructions for testing the installation say:

> change directory into the root folder of the repository

But the installation instructions don't indicate any repository directory. Perhaps add something about where to find this directory for a `conda` installation?

**Comment by bhillmann** [comment]
Does your confusion exist for the development installation? If so, you need to clone the repository first. I updated the README to reflect this.

**Comment by ReneKat** [comment]
Hello!
I am eager to use shogun on my shallow shotgun metagenomics dataset. Currently, I am troubleshooting the
`python -m unittest discover shogun`

I have installed shogun via conda and cloned the git repository. I cd into the repository to run this test and I receive the following error:

> FAIL: test_burst_pipeline (tests.test_pipeline.TestAligner)
> 
> Traceback (most recent call last):
>   File "/mnt/c/Users/Student/github/SHOGUN/shogun/tests/test_pipeline.py", line 93, in test_burst_pipeline
>     self.assertTrue(df_infile.shape[1] == 3)
> AssertionError: False is not true
> 
> Ran 52 tests in 221.048s
> 
> FAILED (failures=1)

When I run `conda uninstall shogun`, it uninstalls dependencies as well. So that when I install via pip the test fails because it cannot find the dependencies. 

Any help would be appreciate.
Many thanks,
Rene

**Comment by mortonjt** [comment]
Can't comment on the above error, but I am noticing machine-specific installation issues.
I'm able to install shogun on my local machine without problems. But I run into issues installing it on my cluster via 

`conda create -n shogun -c knights-lab shogun`

I get the following error
```
conda create -n shogun -c knights-lab shogun
Collecting package metadata (current_repodata.json): done
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                     

UnsatisfiableError: 

```
Not exactly sure what is going on ...

**Comment by GabeAl** [comment]
What version of python is default on the cluster? Might need to create a
new environment with the python version locked in! Does it list what's
unsatisfied? If not it seems like a weird glitch in the default env's
package config

But I'm no conda expert so I'll let Ben weigh in!

**Comment by bhillmann** [comment]
@mortonjt You are probably experiencing this issue because you aren't including the bioconda channel in your installation command.

`conda install -c knights-lab -c bioconda -conda-forge shogun`

**Comment by mortonjt** [comment]
yes, `conda install -c knights-lab -c bioconda -c conda-forge shogun` fixed this issue.  Thanks!

**Comment by GabeAl** [comment]
Thanks Ben!! At least I was right about conda dependencies (but wrong about
the python version thing!). Wonder if there's a way to implicitly add those
repo dependencies into the recipe?

Glad it works for you now, Jamie! I'm curious how/why the home installation
worked with the minimal commandline. Maybe your repofile had those repos in
it already?

Cheerio,
Gabe

**Comment by mortonjt** [comment]
I already had conda-forge in my .condarc so that may have been the reason.
